browser/android/preferences/background_video_playback_tab_helper.cc

```diff
@@ -39,14 +39,11 @@ bool IsBackgroundVideoPlaybackEnabled(content::WebContents* contents) {
     return false;
 
   const std::string host = contents->GetLastCommittedURL().host();
-  const std::string path = contents->GetLastCommittedURL().path();
```

Comment:
I suppose they made changes on YouTube side, and now we need to perform injection before start watching

Reply:
shouldn't we just be doing `net::registry_controlled_domains::SameDomainOrHost` here?